.github/workflows/build-wasm.yml

```diff
@@ -61,9 +61,9 @@ jobs:
       - uses: codex-team/action-nodejs-package-info@v1.1
         id: package
 
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@65c4c4a1ddee5b72f698fdd19549f0f0fb45cf08 # v4.6.0
```

Comment:
why not just `upload-artifact@v4`? Was this failing before?
The same for the rest of the `@commithash`

Reply:
they [triggered](https://github.com/DataDog/dd-wasm-js-rewriter/pull/114#discussion_r1954702263) a Code Vulnerability